src/formats/jsonLd.test.ts

```diff
@@ -20,11 +20,31 @@
 //
 
 import { jest, describe, it, expect } from "@jest/globals";
+import { Response } from "cross-fetch";
 import { foaf, rdf } from "rdf-namespaces";
 import { DataFactory } from "n3";
-import jsonld from "jsonld";
+
+// FIXME: Move this to jest.config.ts when merging with the universal fetch PR
+import type * as RDF from "@rdfjs/types";
+import type { IQuadTerms } from "jest-rdf/lib/matchers/toBeRdfDatasetMatching";
+import matchers from "jest-rdf/lib/matchers";
 import { getJsonLdParser } from "./jsonLd";
 
+interface Matchers<R> {
```

Comment:
I am not sure this is necessary. It looks in the upstream that declaring it in the jest namespace makes it available to the jest globals, but that's not the case here.

Reply:
Yes you're right - was originally doing type casting with this but replaced it with a `@ts-ignore` in the end.